### DISCUSSION TITLE
Accessibility report

### DISCUSSION THREAD

**Discussion by Shaheerlatif1** [GoogleChrome/lighthouse#13891] [General]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### What happened?

There is a sign up page having register link. I clicked register link a pop up is displayed with 3 options and some description. I need to check the accessibility for that pop up. When I generate the accessibility report the page got refresh and the pop up is gone. 

### What did you expect?

Accessibility report should be generated for a pop up window 

### What have you tried?

There is a sign up and register page I clicked register a pop up is displayed with 3 option and some description. I need to check the accessibility for that pop up. When I generate the accessibility report the page got refresh and the pop up is gone. 

### How were you running Lighthouse?

Chrome DevTools..

### Lighthouse Version

9.4.0

### Chrome Version

100.0.4896.127

### Node Version

.

### OS

Windows

### Relevant log output

```shell
There is a sign up and register page I clicked register a pop up is displayed with 3 option and some description. I need to check the accessibility for that pop up. When I generate the accessibility report the page got refresh and the pop up is gone.
```

**Comment by adamraine** [comment]
This looks like the perfect case for [Lighthouse user flows](https://github.com/GoogleChrome/lighthouse/blob/master/docs/user-flows.md)! You can use a snapshot report to analyze accessibility of your popup without reloading the page.

By default, the Lighthouse panel in DevTools will use navigation mode, but we have an experimental feature to enable snapshot mode in DevTools: https://developer.chrome.com/blog/new-in-devtools-101/#lighthouse

**Comment by Shaheerlatif1** [comment]
***@***.***

Hi team,

I’m unable to see the snapshot mode.

**Reply by connorjclark** [reply]
It is currently only available in Chrome canary.

**Reply by adamraine** [reply]
Please read https://developer.chrome.com/blog/new-in-devtools-101/#lighthouse for more info on how to enable the experiment.

**Comment by Shaheerlatif1** [comment]
Hi team I’m still unable to see the snapshot mode even using chrome canary please guide me.

***@***.***

**Reply by adamraine** [reply]
Did you enable the experiment?

From https://developer.chrome.com/blog/new-in-devtools-101/#lighthouse:

> To enable the experiment, enable the Use Lighthouse panel with timespan and snapshot modes checkbox under Settings > Experiments.

**Comment by Shaheerlatif1** [comment]
Unable to find the Settings > Experiments can you please share any screenshots? Where is that option?

***@***.***

**Reply by adamraine** [reply]
Click the cog wheel in the upper right:

<img width="322" alt="Screen Shot 2022-04-25 at 9 59 53 AM" src="https://user-images.githubusercontent.com/6752989/165137549-04d8f632-5bb9-4e55-a16b-1e9f2e90b558.png">

Then go to "Experiments" and ensure the highlighted experiment is checked:

<img width="667" alt="Screen Shot 2022-04-25 at 10 00 39 AM" src="https://user-images.githubusercontent.com/6752989/165137808-a2460b25-98d8-436a-b3c3-7a65ea09c96a.png">

**Comment by Shaheerlatif1** [comment]
Thank you so much team I got it and it is working.
